app/personalised-prevention-platform/2025/10/gauging-intent/index.md

```diff
@@ -0,0 +1,132 @@
+---
+title: "Gauging intent"
+description: "Presented with useful and relevant options, do people feel encouraged to take up activities?"
```

Comment:
Is this lede more like: 'Presented with useful and relevant options, how do we know if someone's actually going to use a service we've shown them.'

Reply:
'something' 😂

Reply:
Presented with useful and relevant options, how do we know if someone's actually going to do something based on what we've shown them.

Reply:
OR

How do we know if someone's actually going to do something based on the options we've shown them?